src/main/java/io/ebean/dbmigration/ddlgeneration/platform/SqlServerDdl.java

```diff
@@ -102,13 +124,24 @@ public String createSequence(String sequenceName, int initialValue, int allocati
   
   @Override
   public String alterColumnDefaultValue(String tableName, String columnName, String defaultValue) {
-
+    // Unfortunately, the SqlServer creates default values with a random name.
+    // You can specify a name in DDL, but this does not work in conjunction with
+    // temporal tables in certain cases. So we have to delete the constraint with
+    // a rather complex statement.
+    StringBuilder sb = new StringBuilder();
     if (DdlHelp.isDropDefault(defaultValue)) {
-      return "alter table " + tableName + " drop constraint df_" + tableName + "_" + columnName;
+      sb.append("delimiter $$\n");
```

Comment:
What do you think about this hack?: I need to execute the next statements in one request.
delimiter is MYSQL specific, but works also on MSSQL due special handling in DDL-script runner.
But this means, that I cannot copy & paste the generated script to SqlStudio and execute it there. (maybe I'll change that and wrap it in an EXEC('....') statement)

Reply:
I think someone in SQL Server land needs a good talking to.  A thought is to create and use migration specific stored procedures for sql server.  That is, ensure those stored procedures are created (prerequisite) ... and use them to do this (and perhaps some other things)

Reply:
This is better than what we have but yes longer term ... I suspect  it might be nicer/cleaner to actually create/ensure we have some migration specific stored procedures [for sql server] that we use to drop default constraints, drop columns etc in sql server.